test/wallet.js

```diff
@@ -26,8 +26,27 @@ describe('wallet', function () {
     lib.after(mockery)
   })
 
+  afterEach(function () {
+    lib.afterEach(mockery)
+  })
+
   it('create', function () {
     const result = lib.createWallet()
     snapshot(this.test.fullTitle(), result)
```

Comment:
Can we change this from `result` to `lib.getInfo(result)`. This way we will have a reference what it was before recovery, so that we know that all properties were updated correctly